### ISSUE TITLE
Issue with ores not being replaced

### ISSUE THREAD

**Issue by ZIsGoated** [SagaDeoMissTeam/Ore-Stages-Reborn#1]
I am not sure if this is a singular isolated issue since as far as I know I'm the only one that has brought this up, but the issue im having is that when I make a ore restriction, it'll work as far as the ore being replaced by the said block I want it to replace. But when it comes to the world generation, the ore will not be replaced and in fact it'll still generate. The scripts, as far as I know, still work since when I place the ore block that's being restricted, it'll quickly go to the block that'll be repaced (f.e: iron ore will change to stone when placed, but iron ore will still generate). any help on this would be appreciated.

**Comment by Icebelly** [comment]
Just here to leave a note that I'm running into a similar issue on 1.19.2, right clicking blocks in the world I've set with OreSpawn will update said blocks but on normal generation these blocks are not being hidden properly. Would similarly appreciate any help.

**Comment by alexcsnow456-beep** [comment]
I'm also currently running into this issue on 1.21.1, my script replaces ores that i place manually but ores that are generated by the world are unaffected by my script, however when i break them in survival mode they drop the intended replacement block instead of the ore so it's clear the script is functioning but there is some kind of visual issue perhaps. if anyone has a fix or could help me it would be greatly appreciated.

**Comment by DeusSixik** [comment]
Yes, it looks like for some reason the world renderer is not being updated. I do not know when I will fix it, as I do not have time.

Sorry

**Comment by alexcsnow456-beep** [comment]
Thanks a lot for the response I've since tried a few other things to fix the issue but no luck so yeah it seems the mod itself is broken, if anyone has any ideas for an alternative or workaround that would be great.
